doc/contributing/docs/style.rst

```diff
@@ -73,22 +170,28 @@ aren't familiar with the
 or don't know the difference between them.
 For this reason, it's best to avoid using them.
 
-Specify link text
------------------
 
-When you provide a :doc:`link </contributing/docs/markup/links>`, clearly specify
-where it leads. In this way, you will not mislead the reader.
+    
+Spelling and punctuation
+------------------------
 
-Bad example:
+US vs British spelling
+~~~~~~~~~~~~~~~~~~~~~~
 
-    For more details, click :doc:`here </contributing/docs/markup/links>`.
+We use the US English spelling.
 
-    Use :doc:`this </contributing/docs/markup/links>`.
+Check your spelling and punctuation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Good example:
+Consider checking spelling, grammar, and punctuation with special tools.
 
-    For more details, refer to the documentation on
-    :doc:`making links </contributing/docs/markup/links>`.
+Consistent ending punctuation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-    Use full :doc:`link names </contributing/docs/markup/links>`.
-    
\ No newline at end of file
+Make sure that your lists and tables have consistent punctuation---either
+all similar list/table items end with a period or they all don't.
+In the example below, *all* items in the second column don't have
+ending punctuation. Meanwhile, *all* items in the fourth column end with a period:
```

Comment:
IFAIU, we should prefer to end a list/table item with a pediod if most of items are a kind of sentence (or even several sentences), not just noun group. Similar to puntuation of lists in Russian. (It is just common sense, I don't know, whether there are grammar rules around this for English.)

What is interesting is when we have a list/table of noun groups (or short phrases). Am I understand correct: we should NOT end an items with a comma, just leave it without punctuation marks? It is different from Russian, but that's what I usually see in English texts. I don't know whether it is normative or informal 'internet style'.

Reply:
In my experience, punctuation rules differ across languages.

For English docs, we've been using recommendations from the Microsoft Style Guide, which I'll cite below. I see no reason to stop relying on them now.

[For lists](https://docs.microsoft.com/en-us/style-guide/scannable-content/lists#punctuation):

In bulleted and numbered lists, end each list item with a period if:
- Any item forms a complete sentence when combined with the list introduction that precedes the colon.
(Exception) Don’t use periods if all items have three or fewer words or if the items are UI labels, headings, subheadings, or strings.
- Any item by itself is a complete sentence.

Don’t use semicolons, commas, or conjunctions (like and or or) at the end of list items.

[For tables](https://docs.microsoft.com/en-us/style-guide/scannable-content/tables#punctuation):

For the text in cells, use periods or other end punctuation only if the cells contain complete sentences or a mixture of fragments and sentences.

Reply:
Thanks! I would like to see an explicit recommendation in the guide, because 1) it is not obvious for me what is normative; 2) we have the section that looks quite relevant.

Reply:
We're planning to do it in https://github.com/tarantool/doc/issues/2028. Is that OK?

Reply:
That's nice, thanks again!

Reply:
Hm. But the updated version states the following:

> For the text in cells, use periods or other end punctuation
> only if the cells contain complete sentences or a mixture of fragments and sentences.

And it is not very clear for me:

1. Whether it is about Russian, English or both (I guess English)?
2. Whether I can use a comma for complete sentences (I guess, no)?
3. If we're about English, I would state explicitly 'don't use a comma or a period when all items are not sentences'. It can be deduced from current wording, but requires some mental effort. Explicit is better than implicit.
4. If we're about Russian, I would state explicitly that formatting (as a list) does not change punctuation rules and the comma should be used between homogenous parts of a sentence.

Sorry for nits.

Reply:
Nits are good when you're making instructions. :)

I tried applying what you suggested in Item 3, but it made me confused. Please take a look at the updated version.

Reply:
What about the case where an item by itself isn't a complete sentence, but it forms a complete sentence when combined with the sentence preceding the list? Looks like it's missing here.

Reply:
We discussed it privately and agreed that both cases looks the same, so there is no sense to make the instruction more complex. @patiencedaur updated the instruction, it looks okay for me (in the sense that I can understand it and follow). So I'll resolve the discussion.